shared/test/resources/tests.yaml

```diff
@@ -1182,11 +1182,15 @@
 
 "ɾ":
   - {in: [10], out: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]}
+  - {in: [[3, 2]], out: [[[1, 1], [1, 2]], [[2, 1], [2, 2]], [[3, 1], [3, 2]]]}
```

Comment:
I'd like to see an example like `in: [["foo", 10, [2, 2]]]` to make sure the vectorisation works properly